distributions/docker/replication/src/main/resources/demo-config/application.properties

```diff
@@ -1,11 +1,8 @@
 logging.level.root=INFO
-
```

Comment:
We shouldn't need to have multiple application properties/yml files. Goes back to the packaging of your application module again :smile: